standards-catalogue/source/Standards/domainspecific/iso20022/index.html.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Is this about the ISO20022 or RFC4180? I don't recall the first being in review by the DSA?

Reply:
I will check with Em - this is just generated from the spreadsheet he sent me...